src/features/deeplinks/routing.test.ts

```diff
@@ -105,7 +108,138 @@ describe('DEEPLINK_TO_SCREEN_CONFIGURATION', () => {
     it('should return Search page when no params are passed', () => {
       const configureScreen = DEEPLINK_TO_SCREEN_CONFIGURATION['recherche']()
       expect(configureScreen.screen).toBe('Search')
-      expect(configureScreen.params).toBe(undefined)
+      expect(configureScreen.params).toEqual({
+        aroundRadius: undefined,
+        beginningDatetime: undefined,
+        date: undefined,
+        endingDatetime: undefined,
+        geolocation: undefined,
+        hitsPerPage: undefined,
+        locationType: undefined,
+        offerCategories: undefined,
+        offerIsDuo: undefined,
+        offerIsFree: undefined,
+        offerIsNew: undefined,
+        offerTypes: undefined,
+        place: undefined,
+        priceRange: undefined,
+        query: undefined,
+        showResults: undefined,
+        tags: undefined,
+        timeRange: undefined,
+        venueId: undefined,
+      })
+    })
+
+    it('should return Search page when params are passed', () => {
+      const screen = 'Search'
+      const stringifier = screenParamsStringifier[screen]
+      const searchState: SearchState = {
+        aroundRadius: 100,
+        beginningDatetime: new Date('2021-08-10T15:02:01.100Z'),
+        date: {
+          option: 'currentWeek' as DATE_FILTER_OPTIONS,
+          selectedDate: new Date('2021-08-10T15:02:01.100Z'),
+        },
+        endingDatetime: new Date('2021-08-10T15:02:01.100Z'),
+        geolocation: {
+          latitude: 48.891304999999996,
+          longitude: 2.3529866999999998,
+        },
+        hitsPerPage: 20,
+        locationType: 'AROUND_ME' as LocationType,
+        offerCategories: [
+          'CINEMA',
+          'VISITE',
+          'MUSIQUE',
+          'SPECTACLE',
+          'LIVRE',
+          'LECON',
+          'FILM',
+          'JEUX_VIDEO',
+          'PRESSE',
+          'CONFERENCE',
+          'INSTRUMENT',
+          'MATERIEL_ART_CREA',
+        ],
+        offerIsDuo: false,
+        offerIsFree: false,
+        offerIsNew: true,
+        offerTypes: {
+          isDigital: true,
+          isEvent: true,
+          isThing: true,
+        },
+        place: {
+          name: {
+            long: 'Rue Victor Hugo, Bordeaux',
+            short: 'Rue Victor Hugo',
+          },
+          extraData: {
+            city: 'Bordeaux',
+            department: 'Gironde',
+          },
```

Comment:
Tips : ce genre d'objet peut tenir sur une ligne si tu le souhaites
```suggestion
          extraData: { city: 'Bordeaux', department: 'Gironde' },
```